content/posts/irods-development-update-september-2021.md

```diff
@@ -0,0 +1,251 @@
+Title: iRODS Development Update: September 2021
+Date: 2021-09-27 08:00
+Author: Kory Draughn
+Slug: irods-development-update-september-2021
+Status: published
+
+
+Hello all! We've reached the end of another month and with that comes exciting iRODS news.
+
+We're excited to report that the [C++ REST API](https://github.com/irods/irods_client_rest_cpp) and [Zone Management Tool (ZMT)](https://github.com/irods/irods_client_zone_management_tool) are approaching their first official release! We have a few more issues to take care of before release, so look for those within a few days/weeks.
+
+The [indexing plugin](https://github.com/irods/irods_capability_indexing) is almost ready to be released. Support for the NIEHS schema has been implemented and tested. It is now in the final stages of polish.
+
+Metalnx has received quite a number of changes in the past month. Search has been reimplemented to use GenQuery. This change allows Metalnx to honor permissions and present only the logical space. The AVU search and Property search have been combined to provide an interface closer to that of iquest. We hope the community finds these changes helpful moving forward.
+
+Jargon is getting support for Replica Access Tokens which enable parallel transfer over multiple streams. Development is going well and we hope to see a release soon. Big thanks to Mike Conway for his work.
+
+Like Jargon, we're working to add support for Replica Access Tokens to the [Globus Connector](https://github.com/irods/irods_client_globus_connector).
```

Comment:
@trel Thoughts on this line?

Reply:
Seems good - anyone interested can click and read more.

Reply:
Cool. I think that's it.